routes/comments.route.js

```diff
@@ -6,7 +6,7 @@ const router = express.Router(); // eslint-disable-line new-cap
 
 router.route('/')
   /** GET /api/comments	 - Get list of comments */
-  .get(passport.authenticate('jwt', { session: false }), commentsController.index)
```

Comment:
Hôm trước biết đại ca phải làm tay to đoạn authenticate rồi, nhưng mà không ngờ là tay phải to đến mức độ này :))

Reply:
cũng có cách chia ra 2 cái ở trong `app.js` 1 cái là những route cần authen, 1 cái là không, nhưng mà a thấy app mình hơi nhỏ, nên a không chia, cho vào trước route luôn 🤕

Reply:
em tưởng là anh làm cái middleware ở giữa check nếu là get request thì k cần `authenticate` chứ ?

Reply:
- Nếu chia ra thì sẽ như cái anh comment bên trên với Tùng, nhưng app m nhỏ nên a cũng không chia.
- Anh ko thấy việc để ai cũng có thể lấy đc data từ server mình là hay, nên GET thì cũng cần token. App mình thì POSTS với COMMENTS không để authenticate thôi, còn USERS vẫn cần authenticate để lấy được data.